packages/rest/src/rest.application.ts

```diff
@@ -4,13 +4,21 @@
 // License text available at https://opensource.org/licenses/MIT
 
 import {Binding, Constructor} from '@loopback/context';
-import {Application, ApplicationConfig, Server} from '@loopback/core';
+import {
+  Application,
+  ApplicationConfig,
+  Server,
+  Component,
+  mountComponent,
+  BindingScope,
+} from '@loopback/core';
 import {OpenApiSpec, OperationObject} from '@loopback/openapi-v3-types';
 import {PathParams} from 'express-serve-static-core';
 import {ServeStaticOptions} from 'serve-static';
 import {format} from 'util';
 import {RestBindings} from './keys';
 import {RestComponent} from './rest.component';
+import {ExplorerComponent} from './explorer.component';
```

Comment:
Don't bundle the explorer component inside `@loopback/rest`. The coupling between the API explorer (e.g. a specific swagger-ui version used) and the runtime (`@loopback/rest`) is the thing we want to avoid by moving explorer into a new component.

The API Explorer should be added to applications as part of our project scaffolding provided by `lb4 app`. Nice to have: `lb4 app` can ask the user whether they want to add API Explorer to their project or not.